syntax = "proto3";

option java_package = "com.terminus.ble.common.protobuf";
package TslBLEProto;


// 开门
message TSLBLEDeviceLOCKRequest {
   int32 security = 1;     
   int32 type=2;
   int64 uid=3;
   int64 timeStamp=4;
}
